### DISCUSSION TITLE
Is that any way to make a group of my project

### DISCUSSION THREAD

**Discussion by LeoYelton** [community/community#38621] [Other Feature Feedback, Questions, & Ideas]
I think as a repository to save code or any other thing , be able to make a directory to save my own fork group or any star group is a basic view. Besides, it is awesome to directly copy whole group from a coding master, right? For example , if i am a junior engineer and i'm wondering which repository to follow or fork. I can directly search "Github/Amazon/Microsoft...etc" senior or expert engineer's favourite bookmark group. And i will be able to learn what expert focus at to ensure my sight keeping latest.

**Comment by mdAshik922** [comment]
I don't know if there is a repository or group of such seniors, it would be better if there is

**Comment by ossanna16** [comment]
Hi there @Yelixing and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
